.github/workflows/links.yml

```diff
@@ -14,11 +14,16 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
+      - name: Remove symlink
```

Comment:
What's the purpose of removing the symlink if we're going to exclude it below?

Reply:
I tried excluding file(docs/source/readme.md) from link checking, but link checker excluding readme.md in both root and also in docs/source , so I tried removing symlink ,then link checker worked. The exclude below is not for excluding file, there may be a reference to this symlink file in other doc, so to ensure that link checker doesn't validate this reference as it is now-nonexistent soft link. Although it will still work without excluding this.

Reply:
Deleting the symlink is definitely a workaround, but not a pretty one. Let's figure out how to do without it, remove it.

You also have `file:///home/runner/work/opensearch-py/opensearch-py/docs/source/README.md` in `.lychee.excludes`, so it sounds like you're saying that it doesn't work?